.github/workflows/tests.yml

```diff
@@ -53,14 +55,51 @@ jobs:
 
       - name: Save Playwright Test HTML report
         if: always()
-        uses: actions/upload-artifact@v2
+        uses: actions/upload-artifact@master
```

Comment:
Please keep the v2, otherwise we might be affected if `master` has bugs or breaking changes

Reply:
hum, got it!
ok, returning the version. 👍

Reply:
@uLucasFraga this adjustment is missing

Reply:
oh gosh!
I missed that haha

my bad and thanks. 🎯